VideoPlayer/VideoPlayer.js

```diff
@@ -527,10 +527,10 @@ const VideoPlayerBase = class extends Component {
 		 * @public
 		 */
 		playbackRateHash: PropTypes.shape({
-			fastForward: PropTypes.arrayOf(PropTypes.string),
-			rewind: PropTypes.arrayOf(PropTypes.string),
-			slowForward: PropTypes.arrayOf(PropTypes.string),
-			slowRewind: PropTypes.arrayOf(PropTypes.string)
+			fastForward: PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.string, PropTypes.number])),
+			rewind: PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.string, PropTypes.number])),
+			slowForward: PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.string, PropTypes.number])),
+			slowRewind: PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.string, PropTypes.number]))
 		}),
 
```

Comment:
Could you please fix L1398

	 * Changes the playback speed via [selectPlaybackRate()]{@link sandstone/VideoPlayer.VideoPlayer#selectPlaybackRate}.
	
==>

	 * Changes the playback speed.

Reply:
It's fixed. Also L1377, L1455 are fixed.